dart/README.md

```diff
@@ -40,10 +40,48 @@ import 'dart:async';
 import 'package:sentry/sentry.dart';
 
 Future<void> main() async {
-  await Sentry.init((options) {
-    options.dsn = 'https://example@sentry.io/add-your-dsn-here';
-  });
+  await Sentry.init(
+    (options) {
+      options.dsn = 'https://example@sentry.io/add-your-dsn-here';
+    }, 
+    initApp, // Init your App.
```

Comment:
I think we should just do the `runApp(MyApp())` here now, having a method that throws and captures is a bit non sense for the appRunner callback

Reply:
there is no `runApp` in Dart. I removed the initApp implementation. Not sure it makes sense to declare an empty method  `Future<void> initApp() async { // your app code }`, so I let `void initApp() { // your app code }` . WDYT ?

Reply:
well this is the README right, the `callback` is mainly used for people that have an UI so `runApp(MyApp())` makes sense, of course on the sample we can't do it otherwise it won't compile

Reply:
this is the dart/readme , and dart web has no runApp either. It only makes sense in flutter/readme ? Am I missing something ?

Reply:
so the `appRunner` callback for the Dart SDK is an optional parameter if they don't want to use the Flutter package for some reason. if they decide to use the `appRunner` callback, that means they have a sort of UI to executed on the `runZoned` method, right, it could be a `runApp(MyApp())` or really any other method, this is just an example that they should do inside of this callback.

Reply:
an empty method + comment // run your app here, also would work btw